docs/UserGuide.md

```diff
@@ -539,50 +472,167 @@ If a parameter is expected only once in the command, but you specified it multip
 **Examples**:
 * Add a meeting and there are no clashes.
     * `meet 1 20.05.2021 15:00 16:00 MRT`
-
       ![meet-add](images/meet-add.png)
-
+      <br><br>
 * Add a meeting but there are clashes.
     * `meet 3 -add 20.05.2021 15:30 17:30 MRT`
-
       ![meet-clash](images/meet-clash.png)
 
+
 * Delete a meeting.
```

Comment:
There are breaklines between the first bullet (Add a meeting and there are no clashes.) and the second (Add a meeting but there are clashes.), but none between the second and third, and third and fourth.